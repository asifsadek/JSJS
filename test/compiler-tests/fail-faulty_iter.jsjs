val l = [1,2,3,4,5];

val sq = /\(x) => x * x;

List.iter(sq, l);