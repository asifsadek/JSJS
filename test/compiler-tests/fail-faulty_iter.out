Type error: expected value of type 'unit', got a value of type 'num' instead
